webapp/app.py

```diff
@@ -25,42 +28,49 @@
 )
 
 app.secret_key = os.environ["SECRET_KEY"]
+
 open_id = OpenID(
     stateless=True, safe_roots=[], extension_responses=[MacaroonResponse]
 )
 
 
+def _api_request(url_path, method="GET", params=None, json=None, headers=None):
+    """
+    Make API calls to the anbox API, passing any 401 errors to Flask to handle
+    """
+
+    response = session.request(
+        method,
+        url_join(ANBOXCLOUD_API_BASE, url_path),
```

Comment:
Just a naive question. Is there a reason we don't use a f-string? Since the 2 variables are strings. Is it for readability?

``` py
f"{ANBOXCLOUD_API_BASE}{url_path}"
```